components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -447,6 +466,46 @@ - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentConta
   [self.view setNeedsLayout];
 }
 
+- (void)expandToFullScreenWithDuration:(NSTimeInterval)duration {
+  CGFloat topSafeArea = 0;
+  if (@available(iOS 11.0, *)) {
+    topSafeArea = self.view.safeAreaInsets.top;
+  }
+  CGFloat totalHeight = self.headerViewController.preferredContentSize.height +
```

Comment:
can you refrain from using the preferredContentSize of the contentViewController? Is it the same as the cached one or has it changed at this moment? have a look at `_contentVCPreferredContentSizeHeightCached`